acceptance/tests/pxp-module-puppet/run_puppet_result_failed.rb

```diff
@@ -0,0 +1,53 @@
+require 'pxp-agent/test_helper.rb'
+require 'puppet/acceptance/environment_utils'
+
+test_name 'C93063 - pxp-module-puppet run expect puppet failure' do
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  app_type = File.basename(__FILE__, '.*')
```

Comment:
Does this get the filename, so we can name the temp environment after the test name? It might be useful to add a code comment here; or rename app_type to something like "test_file_name"

I think (at least irb allows this) you can also simply refer to $0 (arguable if that is more or less readable though)